.github/workflows/e2e-feature-tests.yml

```
@@ -116,6 +116,14 @@ jobs:
       run: |
         .github/scripts/e2e_tests_brownfield_use_case.sh
 
+    # the script accepts number of requests for  APIs for CREATE_BUCKET, DELETE_BUCKET, GET_INFO
```

```suggestion
    # the script accepts number of requests for APIs for CREATE_BUCKET, DELETE_BUCKET, GET_INFO
```
or is there a missing word ?